app/views/dashboard_v2/_sidebar.html.erb

```diff
@@ -55,7 +55,20 @@
     </div>
   </div>
 </div>
-
+<div>
+<% if !@recommendations.nil? %>
```

Comment:
```suggestion
<% @recommendations ||= Tag.get_recommendations("water-quality", "air-quality") %>
```

Reply:
@jywarren, I tried this out, but it still doesn't seem to work as I am unable to find cards as expected on the dashboard.